### PR TITLE
Adding Phalcon3 and FlightPHP examples

### PR DESCRIPTION
**Pull request by zQueal**
Self explanatory.

### CODE REVIEW

File: flight/Caddyfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+:80 {
```
Comment (mholt) [inline flight/Caddyfile]: I'll accept this example as soon as the example is for a secure configuration. ;)

**Comment by zQueal** [comment]
localhost:8080
    proxy /isso 127.0.0.1:8080
    proxy / mysite.example.com:3333
    http://localhost:8080
    proxy / localhost:8000
    
I can't see where you've enforced that restriction with any other pull request until now... Almost every single one of the 10 or so I checked (to see if I missed that requirement) I see non-standard non-https ports. Seems weird to start instituting it with a random PR.

Accept the changes or don't. I have them for my reference, I pushed them here to help others.

**Comment by mholt** [comment]
@zQueal Pardon, but all those examples are proxying to localhost-only backends, and that first one is serving only on localhost. Your example serves on `:80` which opens up a site to every interface -- including public ones -- and should be secured in order to be acceptable for this repository.

Don't be offended by a code review. This is business as usual, nothing personal.

**Comment by francislavoie** [comment]
This is definitely a useful example to have, and I'd also love to see it merged in. As a PHP dev who's starting to dabble with Phalcon, this is nice to see. @zQueal could you please just change `:80` to `localhost:80`? That would be enough to merge it in. +1 to @mholt's comment, your example is by default insecure for development in most situations, so more secure defaults are preferred.

**Pull request closed without merging**